lib/toggle-quotes.coffee

```diff
@@ -26,25 +29,30 @@ toggleQuoteAtPosition = (editor, position) ->
   prefix = ''
   [prefix, quoteCharacter] = text if /[uUr]/.test(quoteCharacter)
 
-  oppositeQuoteCharacter = getOppositeQuote(quoteCharacter)
+  nextQuoteCharacter = getNextQuote(quoteCharacter, quoteChars)
+  return unless nextQuoteCharacter
   quoteRegex = new RegExp(quoteCharacter, 'g')
   escapedQuoteRegex = new RegExp("\\\\#{quoteCharacter}", 'g')
-  oppositeQuoteRegex = new RegExp(oppositeQuoteCharacter, 'g')
+  nextQuoteRegex = new RegExp(nextQuoteCharacter, 'g')
 
   newText = text
-    .replace(oppositeQuoteRegex, "\\#{oppositeQuoteCharacter}")
+    .replace(nextQuoteRegex, "\\#{nextQuoteCharacter}")
     .replace(escapedQuoteRegex, quoteCharacter)
-  newText = prefix + oppositeQuoteCharacter + newText[(1+prefix.length)...-1] + oppositeQuoteCharacter
+  newText = prefix + nextQuoteCharacter + newText[(1+prefix.length)...-1] + nextQuoteCharacter
 
   editor.setTextInBufferRange(range, newText)
 
-getOppositeQuote = (quoteCharacter) ->
-  if quoteCharacter is '"'
-    "'"
-  else
-    '"'
+getNextQuote = (c, cs) ->
```

Comment:
Done, but fwiw I find `c` and `cs` much easier to distinguish from one another than `quoteCharacter` and `quoteCharacters`